### ISSUE TITLE
Slow Chunk Loading

### ISSUE THREAD

**Issue by nemes93** [AlexModGuy/Citadel#164]
In version 1.20.1 having citadel between ur mods make the chunks load SUPER slow, making the game experience a little bit complicated.
The problem here is that this mod is a dependency for a lot more mods.
Idk if this problem can be fixed

**Comment by Veynam** [comment]
I know this is very old but I believe im also experiencing the same issue after an update

**Comment by Veynam** [comment]
Also facing this issue with latest release. Strangely it only seems apear old worls

**Comment by MrBOOMberMAN** [comment]
same problem here

**Comment by kinnikinnick** [comment]
Is this still a problem in 2.6.0 for 1.20.1?  I have a server that's heavily modded and it runs perfectly except for chunk *generation*. It was generating well early on but not it's gotten to the point where it take 30+ seconds to generate just a few chunks.  I'm using Alex's Caves (and Mobs) so I think it would require a world reset if I wanted to test this yes?  I can't just remove Caves without causing issues if the caves and structures are already generated I assume.  Is there a way for me to test if this is the issue or fix this if it is?

**Comment by Veynam** [comment]
Yes it's still a problem and it feels like it's not being addressed at all.
There's no way to test this as far as I'm aware because the problem
seemingly only occurs on worlds that have been running for a long time.
Please correct me if i'm wrong. You cannot remove the mod on an existing
world.

**Comment by kinnikinnick** [comment]
I asked on the discord and I don't think I got a response.  There was one person who had the same issue just over a year ago and nobody responded to her either.  I tried to remove the mod from my server but I couldn't get the world to generate properly anymore so i just ended up starting a new world.. 😣 it's very disappointing as Alex's Mobs and Caves are two of my most anticipated mods to try with friends.  Generation is much better though.

**Comment by Veynam** [comment]
This helped me fix it manually.

https://github.com/AlexModGuy/Citadel/issues/101#issuecomment-1566345740

 Its a bit complicated but im sure you can figure it out. Seems to have
fixed slow world gen on my servers 9 month old world

**Comment by kinnikinnick** [comment]
I thought this would only work if you had a version of a Citadel to upgrade to that didn't have the issue?  I looked at downgrading but it seemed all the versions of Alex's Caves required 2.6.0..

**Comment by Veynam** [comment]
I started with a much older version of citadel and all the mods requiring
it (was newest at the time). In my case the issue popped up because of
updating citadel on an existing world. The solution I posted worked for my
case. Now a new chunk takes maybe a second to generate where before it was
a good 10 seconds per chunk if not more.
